admin-frontend/open_admin_app/lib/widgets/admin_api_keys/list_admin_api_keys_widget.dart

```diff
@@ -0,0 +1,317 @@
+                  showCheckboxColumn: false,
+                  sortAscending: sortToggle,
+                  sortColumnIndex: sortColumnIndex,
+                  columns: [
+                    DataColumn(
+                        label: const Text('Name'),
+                        onSort: (columnIndex, ascending) {
+                          onSortColumn(snapshot.data!, columnIndex, ascending);
+                        }),
+                    DataColumn(
+                      label: const Text('Groups'),
+                      onSort: (columnIndex, ascending) {
+                        onSortColumn(snapshot.data!, columnIndex, ascending);
+                      },
+                    ),
+                    DataColumn(label: const Padding(
+                      padding: EdgeInsets.only(left:12.0),
+                      child: Text('Actions'),
+                    ), onSort: (i, a) => {}),
+                  ],
+                  rows: [
+                    for (Person p in snapshot.data!)
+                      DataRow(
+                          cells: [
+                            DataCell(Text(
+                                    p.name!,
+                                  )),
+                            DataCell(Text('${p.groups.length}')),
+                            DataCell(Row(children: <Widget>[
+                              Tooltip(
+                                message: "bla",
+                                child: FHIconButton(
+                                  icon: const Icon(Icons.info),
+                                  onPressed: () => bloc.mrClient
+                                      .addOverlay((BuildContext context) {
+                                    return ListUserInfoDialog(bloc, p);
+                                  }),
+                                ),
+                              ),
+                              FHIconButton(
+                                  icon: const Icon(Icons.edit),
+                                  onPressed: () => {
+                                        ManagementRepositoryClientBloc.router
+                                            .navigateTo(context, '/edit-admin-api-key',
+                                                params: {
+                                              'id': [p.id!.id]
+                                            })
+                                      }),
+                              // const SizedBox(
+                              //   width: 8.0,
+                              // ),
+                              FHIconButton(
+                                icon: const Icon(Icons.delete),
+                                onPressed: () => bloc.mrClient
+                                    .addOverlay((BuildContext context) {
+                                  return bloc.mrClient.person.id!.id ==
+                                          p.id!.id
+                                      ? cantDeleteYourselfDialog(bloc)
```

Comment:
this check shouldn't be necessary as an Admin Service Account is never using the UI, so they aren't deleting themselves